types/react-svg-radar-chart/index.d.ts

```diff
@@ -0,0 +1,55 @@
+// Type definitions for react-svg-radar-chart 1.2
+// Project: https://github.com/Spyna/react-svg-radar-chart
+// Definitions by: Lukas Tutkus <https://github.com/luksys5>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Typescript Version 3.5
+// Minimum TypeScript Version: 3.0
```

Comment:
Why both (both works btw)? Is the minimum 3.5?
```suggestion
```

Reply:
Actually I've only tested on 3.5 so I should remove the minimum. Thought that it was not that different